addons/payment/static/src/js/checkout_form.js

```diff
@@ -9,8 +9,8 @@ odoo.define('payment.checkout_form', require => {
         selector: 'form[name="o_payment_checkout"]',
         events: Object.assign({}, publicWidget.Widget.prototype.events, {
                 'click div[name="o_payment_option_card"]': '_onClickPaymentOption',
-                'click a[name="o_payment_icon_more"]': '_onClickMorePaymentIcons',
```

Comment:
Keep it like this because we'll soon have payment methods in the cards and payment icons above the "click more" button.